lib/iris/tests/integration/experimental/geovista/test_cube_to_poly.py

```diff
@@ -0,0 +1,66 @@
+# Copyright Iris contributors
+#
+# This file is part of Iris and is released under the BSD license.
+# See LICENSE in the root of the repository for full licensing details.
+"""Integration tests for the `iris.experimental.geovista.cube_to_polydata` function."""
+
+import numpy as np
+
+from iris import load_cube
+from iris.experimental.geovista import cube_to_polydata
+from iris.experimental.ugrid import PARSE_UGRID_ON_LOAD
+from iris.tests import get_data_path
+
+
+def test_integration_2d():
+    file_path = get_data_path(
+        [
+            "NetCDF",
+            "ORCA2",
+            "votemper.nc",
+        ]
+    )
+    with PARSE_UGRID_ON_LOAD.context():
+        global_cube = load_cube(file_path, "votemper")
+
+    polydata = cube_to_polydata(global_cube[0, 1, :])
+
+    assert polydata.GetNumberOfCells() == 26640
+    assert polydata.GetNumberOfPoints() == 26969
+
+
+def test_integration_1d():
```

Comment:
I've done some rough plotting to demonstrate this is doing the right thing:

### GeoVista

<img width="247" alt="Screenshot 2024-03-27 151038" src="https://github.com/SciTools/iris/assets/40734014/62dd96b7-1d63-48b1-a645-c02be9e3dfa2">

### Matplotlib (for comparison)

<img width="205" alt="Screenshot 2024-03-27 150823" src="https://github.com/SciTools/iris/assets/40734014/49c24fa7-d442-4c11-926e-440b2e4334ff">